modules/virt-accessing-rdp-console.adoc

```diff
@@ -6,71 +6,101 @@
 [id="virt-accessing-rdp-console_{context}"]
 = Connecting to a Windows virtual machine with an RDP console
 
-The Remote Desktop Protocol (RDP) provides a better console experience for
-connecting to Windows virtual machines.
-
-To connect to a Windows virtual machine with RDP, specify the IP address of the
-attached L2 NIC to your RDP client.
+Create a Kubernetes `Service` object to connect to a Windows virtual machine (VM) by using your local Remote Desktop Protocol (RDP) client.
 
 .Prerequisites
 
-* A running Windows virtual machine with the QEMU guest agent installed. The
-`qemu-guest-agent` is included in the VirtIO drivers.
-* A layer 2 NIC attached to the virtual machine. 
-* An RDP client installed on a machine on the same network as the
-Windows virtual machine.
+* A running Windows virtual machine with the QEMU guest agent installed. The `qemu-guest-agent` is included in the VirtIO drivers.
+* An RDP client installed on a machine on the same network as the Windows VM.
```

Comment:
Now when we switched to Services, it is not required to be on the same network anymore. We can drop the second part of of this bullet point.